brench/brench.py

```diff
@@ -46,6 +47,16 @@ def run_pipe(cmds, input, timeout):
             proc.kill()
 
 
+
+def compare_output(o1, o2):
+    def my_compare(x, y):
+        try:
+            return abs(float(x)-float(y)) < .0000000001
```

Comment:
Perhaps turn this threshold into a constant, `EPSILON`?

Reply:
good call